src/util/lang_string.js

```diff
@@ -43,12 +43,12 @@
   /**
    * Divide a string in the user perceived single units
    * @memberOf fabric.util.string
-   * @param {String} string String to escape
+   * @param {String} textstring String to escape
    * @return {Array} array containing the graphemes
    */
   function graphemeSplit(textstring) {
-    var i = 0, graphemes = [];
-    for (var i = 0, chr; i < textstring.length; i++) {
+    var i = 0, chr, graphemes = [];
+    for (i = 0, chr; i < textstring.length; i++) {
```

Comment:
if chr is declared outside, inside the for braket is not needed.
Also please, since you are on this thing, rename chr to _chr, because in the ES4 specs chr was a reserved word, and some compilers complain about using it.